.github/workflows/run-test-cases.yml

```diff
@@ -232,6 +232,10 @@ jobs:
         name: Install Kubernetes
         run: |
           sudo apt-get update -y
+          sudo apt-get install -y apt-transport-https ca-certificates curl
```

Comment:
seems fine and correct, but strange that we suddenly need it after all this time?

Reply:
I wonder if git hub slimmed down their runner image. The image may have previously had these packages